src/modules/form/components/DynamicForm.tsx

```diff
@@ -203,17 +205,19 @@ const DynamicForm = forwardRef(
               </Stack>
             </Grid>
           )}
-          {renderFields({
-            itemChanged: handleChangeCallback,
-            severalItemsChanged: handleChangeCallback,
-            errors: errorState.errors,
-            warnings: errorState.warnings,
-            horizontal,
-            pickListRelationId,
-            warnIfEmpty,
-            locked,
-            visible,
-          })}
+          <DynamicFormContext.Provider value={{ getCleanedValues, definition }}>
```

Comment:
@ZachCMP  adding a provider here so that child components can access the entire form values if necessary. MCI clearance, for example, needs to access a bunch of the current values in order to perform a search against an external system. I expect this to be used rarely.